components/script/dom/document.rs

```diff
@@ -220,6 +222,7 @@ pub struct Document {
     scripts: MutNullableJS<HTMLCollection>,
     anchors: MutNullableJS<HTMLCollection>,
     applets: MutNullableJS<HTMLCollection>,
+    style_shared_lock: StyleSharedRwLock,
```

Comment:
It'd be nice to have a description or a pointer to the rest of the documentation for this.